javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,29 @@
+// Giphy API Key: Wyd7tK4cooBne57SnMhnvk2t813zedju
+
+const btn = document.querySelector("#searchBtn");
+
+const getGif = () => {
+  const gifNumbInput = document.querySelector("#gifNumbInput").value;
```

Comment:
As html element has an `id`, it is better to use `getElementById` method for grabbing it.
`document.getElementById('gifNumbInput')`